user-interface/src/main/java/life/qbic/datamanager/views/projects/project/rawdata/RawDataDetailsComponent.java

```diff
@@ -76,10 +78,22 @@ public RawDataDetailsComponent(@Autowired RawDataService rawDataService,
    *                   be filtered for
    */
   public void setSearchedRawDataValue(String searchTerm) {
+    if (!this.searchTerm.equals(searchTerm)) {
+      refreshGrids();
+    }
```

Comment:
So you only refresh in case the search term changed? If so, you only need to set the search term if it is different as well.